core/src/main/java/org/apache/accumulo/core/spi/compaction/DefaultCompactionPlanner.java

```diff
@@ -225,6 +225,10 @@ private void determineMaxFilesToCompact(InitParameters params) {
     }
   }
 
+  public int getMaxFilesToCompact() {
+    return this.maxFilesToCompact;
+  }
```

Comment:
Is this SPI addition necessary to implement the bug fix? It would good to try to avoid the change in a user-facing component in a bug fix release. We can be looser about it with SPI rather than public API, if necessary, but if it's not necessary, it'd be good to keep it the same.

Reply:
I refactored my test to allow for a compaction example that does not require the accessor method. 
There are now, no changes to DefaultCompactionPlanner.java